src/app/give-help/offer-form/offer-form.component.ts

```diff
@@ -13,6 +9,24 @@ export interface Offer {
 })
 export class OfferFormComponent {
   data = defaults<Offer>();
```

Comment:
You declar data as `Offer` but also assign to it properties from `TempTransportOfferDefinitionDTO`

Reply:
Spłaszczyłem komponenty forma w give-help